arithmetization/src/main/java/net/consensys/linea/zktracer/module/blockdata/BlockdataOperation.java

```diff
@@ -237,8 +233,17 @@ public void trace(Trace.Blockdata trace) {
           .isChainid(opCode == OpCode.CHAINID)
           .isBasefee(opCode == OpCode.BASEFEE)
           .inst(UnsignedByte.of(opCode.byteValue()))
-          .coinbaseHi(coinbaseAddress.slice(0, 4).toLong())
-          .coinbaseLo(coinbaseAddress.slice(4, LLARGE))
+          .coinbaseHi(
+              hub.blockStack()
+                  .getBlockByRelativeBlockNumber(relBlock)
+                  .coinbaseAddress()
+                  .slice(0, 4)
+                  .toLong())
+          .coinbaseLo(
+              hub.blockStack()
+                  .getBlockByRelativeBlockNumber(relBlock)
+                  .coinbaseAddress()
+                  .slice(4, LLARGE))
```

Comment:
We have some code duplication. Also in `TransactionProcessingMetadata.java` we just write `hub.getCoinbaseAddress()`.